### PR TITLE
Adjusts oozelings to use slime speaking verbs

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Changes the oozeling class to use the ask and yell speaking verbs from slimes.

Also comes with a free refactor, for some reason tounges couldnt modify your ask yell or exclaim verbs so i added that too

## Why It's Good For The Game
Consistency with the slimes mob, ic it makes no sense that you stop blorbling messages when asking a question.

The tounge refactor allows for more creativity with tounge types, with 3 new verbs to define

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
tweak: oozelings have 50% more blorbles
refactor: tounges can now modify your ask, yell, and exclaim verbs
/:cl:

### CODE REVIEW

**Comment by ivanmixo** [comment]
Hope you got permission for a species change

**Comment by etherware-novice** [comment]
permission?

**Comment by Fronsis** [comment]
> permission?

Any changes to species need to get an approval before hand iirc

File: code/__DEFINES/text.dm [hunk 1]
```diff
@@ -7,3 +7,6 @@
 #define CENTER(text) {"<center>[##text]</center>"}
 
 #define SANITIZE_FILENAME(text) (GLOB.filename_forbidden_chars.Replace(text, ""))
+
+// for things that could be a list or a str, if its a list return rand element  - candy/ether
+#define PICK_ONE(x) islist(x) ? pick(x) : x
```
Comment (PowerfulBacon) [inline code/__DEFINES/text.dm]: Pick actually works on parameters as well as lists, meaning pick("hello") will always return hello, removing the need for this.

Comment (etherware-novice) [reply code/__DEFINES/text.dm]: ..ah ok i overcompensated after assigning a list to say_verb by accident

File: code/modules/surgery/organs/tongue.dm [hunk 2]
```diff
@@ -314,7 +317,10 @@
 /obj/item/organ/tongue/slime
 	name = "slimey tongue"
 	desc = "It's a piece of slime, shaped like a tongue."
-	say_mod = "blorbles"
+	say_mod = list("blorbles", "bubbles")
+	ask_mod = "inquisitively blorbles"
+	yell_mod = "blorble screams"
```
Comment (PowerfulBacon) [inline code/modules/surgery/organs/tongue.dm]: Blorble screams?

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/__DEFINES/text.dm [hunk 3]
```diff
@@ -7,3 +7,5 @@
 #define CENTER(text) {"<center>[##text]</center>"}
 
 #define SANITIZE_FILENAME(text) (GLOB.filename_forbidden_chars.Replace(text, ""))
+
```
Comment (itsmeow) [inline code/__DEFINES/text.dm]: Why?

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/__DEFINES/text.dm (hunk 3 above)
Comment (etherware-novice) [reply code/__DEFINES/text.dm]: i added a macro bc i didnt know how pick() worked, just ignore

Comment (itsmeow) [reply code/__DEFINES/text.dm]: remove the newlines

**Review by itsmeow — APPROVED** [review]

**Review by itsmeow — DISMISSED** [review]

File: code/modules/mob/living/carbon/human/say.dm [hunk 4]
```diff
@@ -1,7 +1,10 @@
 /mob/living/carbon/human/say_mod(input, list/message_mods = list())
 	var/obj/item/organ/tongue/T = getorganslot(ORGAN_SLOT_TONGUE)
 	if(T)
-		verb_say = T.say_mod
+		verb_say = pick(T.say_mod)
+		verb_ask = pick(T.ask_mod)  // cant believe nobody coded this yet - ether
```
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/say.dm]:
```suggestion
		verb_ask = pick(T.ask_mod)
```

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**